language/analyze.v1beta2.js

```diff
@@ -230,77 +242,73 @@ function analyzeSyntaxInFile (bucketName, fileName) {
   // [END language_syntax_file]
 }
 
-function analyzeEntitySentimentOfText (text) {
-  // [START language_entity_sentiment_string]
+function classifyTextOfText (text) {
+  // [START language_classify_string]
   // Imports the Google Cloud client library
-  const language = require('@google-cloud/language').v1beta2();
+  const Language = require('@google-cloud/language');
+
+  // Creates a client
+  const language = Language.v1beta2();
```

Comment:
Just wondering what's the difference between these two ways of creating a client?

```
const Language = require('@google-cloud/language');
const language = Language({ apiVersion: 'v1beta2' });
```
vs
```
const Language = require('@google-cloud/language');
const language = Language.v1beta2();
```

Reply:
The first one uses the generated v1 client, but points it add the v1beta2 service endpoint. The second one uses the generated v1beta2 client (which points at the `v1beta2` service endpoint by default).